CRM/Upgrade/Incremental/sql/5.56.alpha1.mysql.tpl

```diff
@@ -1 +1,14 @@
 {* file to handle db changes in 5.56.alpha1 during upgrade *}
+
+-- Add in missing indian states as per iso-3166-2
+SELECT @indianCountryID := id FROM civicrm_country WHERE name = 'India' AND iso_code = 'IN';
+INSERT INTO civicrm_state_province (country_id, abbreviation, name) VALUES
+ (@indianCountryID, "LA", "Ladākh"),
+ (@indianCountryID, "DH", "Dādra and Nagar Haveli and Damān and Diu");
+
+SELECT @DHStateID := id from civicrm_state_province WHERE country_id = @indianCountryID ND abbreviation = 'DH';
```

Comment:
@seamuslee001 there is a typo here (ND instead of AND)

`SELECT @DHStateID := id from civicrm_state_province WHERE country_id = @indianCountryID ND abbreviation = 'DH';`

should be

`SELECT @DHStateID := id from civicrm_state_province WHERE country_id = @indianCountryID AND abbreviation = 'DH';
`